test/unit/eigensolver/test_gen_eigensolver.cpp

```diff
@@ -134,7 +134,7 @@ void testGenEigensolver(const blas::Uplo uplo, const SizeType m, const SizeType
 
   const TileElementSize block_size(mb, mb);
 
-  auto create_reference = [&]() -> auto {
+  auto create_reference = [&]() -> auto{
```

Comment:
not sure if you can control this

```suggestion
  auto create_reference = [&]() -> auto {
```

(in case, there is a similar thing in `test_eigensolver`)